.vscode/c_cpp_properties.json

```diff
@@ -30,7 +30,8 @@
       "name": "Quest",
       "cStandard": "c11",
       "cppStandard": "c++20",
-      "intelliSenseMode": "linux-clang-arm64"
+      "intelliSenseMode": "linux-clang-arm64",
+      "compileCommands": "${workspaceFolder}/build/compile_commands.json"
```

Comment:
can we not lol, this breaks my intellisense every time on files that have yet to ever be compiled (annoying af)